core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -1639,6 +1641,29 @@ public void updateViewLocationConflict() {
         .hasMessageContaining("View does not exist: ns.view");
   }
 
+  @Test
+  public void loadViewMetadataTable() {
+    TableIdentifier identifier = TableIdentifier.of("ns", "view");
+
+    if (requiresNamespaceCreate()) {
+      catalog().createNamespace(identifier.namespace());
+    }
+
+    assertThat(catalog().viewExists(identifier)).as("View should not exist").isFalse();
+
+    View view =
+        catalog()
+            .buildView(identifier)
+            .withSchema(SCHEMA)
+            .withDefaultNamespace(identifier.namespace())
+            .withQuery("trino", "select * from ns.tbl")
+            .create();
+
+    assertThat(catalog().viewExists(identifier)).as("View should exist").isTrue();
+    Table table = tableCatalog().loadTable(TableIdentifier.of("ns", "view", "version"));
+    assertThat(table).isInstanceOf(ViewVersionTable.class);
```

Comment:
this here should probably also have some additional checks

Reply:
Check the schema and properties